doc/rst/source/contour.rst

```diff
@@ -33,6 +37,7 @@ Synopsis
 [ |SYN_OPT-b| ]
 [ |SYN_OPT-d| ]
 [ |SYN_OPT-e| ]
+[ |SYN_OPT-f| ]
 [ |SYN_OPT-h| ]
 [ |SYN_OPT-i| ]
 [ |SYN_OPT-l| ]
```

Comment:
-s should be added here and in contour_common.rst_ for consistency.

Reply:
Done